pkg/ctl/create/cluster.go

```diff
@@ -75,7 +75,7 @@ func createClusterCmd(rc *cmdutils.ResourceCmd) {
 			api.SubnetTopologyPublic:  fs.StringSlice("vpc-public-subnets", nil, "re-use public subnets of an existing VPC"),
 		}
 		fs.StringVar(&params.kopsClusterNameForVPC, "vpc-from-kops-cluster", "", "re-use VPC from a given kops cluster")
-		fs.StringVar(cfg.VPC.NAT.Gateway, "vpc-nat-mode", "single", "VPC NAT mode, valid options: HighlyAvailable, Single, Disable")
+		fs.StringVar(cfg.VPC.NAT.Gateway, "vpc-nat-mode", api.ClusterSingleNAT, "VPC NAT mode, valid options: HighlyAvailable, Single, Disable")
```

Comment:
Ah, I see, so we just mixed up casing on review! :)

Reply:
Looks like I missed one, my bad.

Reply:
It was very minor, it happens :)